docs/reference/release-notes/7.10.asciidoc

```diff
@@ -3,6 +3,19 @@
 
 Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
 
+[[known-issues-7.10.0]]
+[discrete]
+=== Known issues
+
+* SQL: If a WHERE clause contains at least two relational operators joined by
```

Comment:
Small nit. I'd apply this across all the changes.
```suggestion
* SQL: If a `WHERE` clause contains at least two relational operators joined by
```

Reply:
Applied, thanks!